CI/flatpak/com.obsproject.Studio.json

```diff
@@ -263,6 +263,32 @@
         }
       ]
     },
+    {
```

Comment:
Please add a paragraph at the commit message explaining why you needed to explicitly add PipeWire here - I'm sure we'll forget the reasons if this is not documented in the commit!